stories/core_components/GeoPushpinChart.tsx

```diff
@@ -68,6 +68,20 @@ storiesOf("Core components/GeoPushpinChart", module)
             }),
         ),
     )
+    .add("with location and viewport", () =>
```

Comment:
`with location and viewport` -> `with North America viewport`

Reply:
updated